vnext/ReactWindowsCore/utilities.h

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+#include <type_traits>
+#include <utility>
+
+namespace facebook {
+namespace react {
+namespace utilities {
```

Comment:
Tests for functions under this namespace will be checked in as a separate pull request.